### PR TITLE
Feature set max steps in constructor

### PR DESCRIPTION
**Pull request by num8er**
There was an issue about setting `maxSteps` globally.

The most popular solution was to modify package files, and set it inside lib/config.js.

But what if user will update packages, or what if it will be used in container or many other reasons that says that to change lib/config.js is not good idea.

Yes, I know that we can set it every time when we call `runActions` method, but why?

After [this](http://stackoverflow.com/questions/40622631/changing-wit-ai-default-max-steps) talk  in StackOverflow - I came to idea to check if there is a feature to set it in constructor.

Seems like was not.

### CODE REVIEW

File: lib/wit.js [hunk 1]
```diff
@@ -210,7 +210,10 @@ const validate = (opts) => {
   if (opts.actions) {
     opts.actions = validateActions(opts.logger, opts.actions);
   }
-
+  if(!opts.maxSteps || !Number.isInteger(opts.maxSteps)) {
```
Comment (stopachka) [inline lib/wit.js]: this is not necessary, as the maxSteps in this.config is already enforced into being a number here -> 

https://github.com/num8er/node-wit/blob/363a23dccc5f04eb6068ce98e171c1db256989ca/lib/wit.js#L213

File: lib/wit.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ function Wit(opts) {
 
   this.runActions = function(sessionId, message, context, maxSteps) {
     if (!actions) throwMustHaveActions();
-    const steps = maxSteps ? maxSteps : DEFAULT_MAX_STEPS;
+    const steps = maxSteps || this.config.maxSteps;
```
Comment (stopachka) [inline lib/wit.js]: nit: would change this to -> 

```
const steps = Number.isInteger(maxSteps) ? maxSteps : this.config.maxSteps
```

**Review by stopachka — CHANGES_REQUESTED** [review]

**Comment by num8er** [comment]
@stopachka please review

**Comment by l5t** [comment]
Closing since we deprecated Stories https://medium.com/wit-ai/launching-built-in-nlp-for-messenger-and-sunsetting-bot-engine-beta-46e9038869a5

**Pull request closed without merging**